lib/config/options/index.ts

```diff
@@ -170,6 +170,18 @@ const options: RenovateOptions[] = [
       type: 'string',
     },
   },
+  {
+    name: 'statusCheckNames',
+    description: 'Custom strings to use as status check names',
```

Comment:
```suggestion
    description: 'Custom strings to use as status check names.',
```

End sentence with a full stop.